qualysapi/api_actions.py

```diff
@@ -1,9 +1,11 @@
+from __future__ import absolute_import
 from lxml import objectify
```

Comment:
For what I can see comparing your fork directly to the original qualysapi, the one file that contains all of your changes is this one, `qualysapi/api_actions.py`, then `qualysapi/util.py`, `qualysapi/config.py` and `qualysapi/settings.py`. (https://github.com/paragbaxi/qualysapi/compare/master...austin-taylor:4.1.1-develop)

As now I am merging the changes with your branch, it should be okay because all of the code you did should stay without issues.